spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/instrument/messaging/TraceMessagingAutoConfiguration.java

```diff
@@ -171,6 +181,28 @@ TracingJmsBeanPostProcessor tracingJmsBeanPostProcessor(BeanFactory beanFactory)
 
 	}
 
+	@Configuration
+	@ConditionalOnProperty(value = "spring.sleuth.messaging.sqs.enabled", matchIfMissing = true)
+	@ConditionalOnClass(QueueMessageHandler.class)
+	@AutoConfigureAfter(TraceSpringMessagingAutoConfiguration.class)
```

Comment:
I don't think `@AutoConfigureAfter` has any effect here since I don't think `SleuthSqsConfiguration` itself is an auto-configuration class, but I could be wrong. I think the `@AutoConfigureAfter` would need to be on the top-level class, which is registered as the auto-configuration class.

Reply:
👍